### ISSUE TITLE
VASP `RunType` is missing `PBE_64`/`LDA_64`

### ISSUE THREAD

**Issue by janosh** [materialsproject/emmet#1101]
https://github.com/materialsproject/emmet/blob/a917fc1cbec99524498943e39f8546008cf9b2db/emmet-core/emmet/core/vasp/calc_types/enums.py#L9-L20

tagging @esoteric-ephemera @tsmathis

**Comment by janosh** [comment]
nvm, just realized `PBE_54` also doesn't exist. maybe it makes more sense to add a `PotcarVersion` enum to `pymatgen` then? specifically `src/pymatgen/io/vasp/inputs.py`. e.g.


```py
@unique
class PotcarVersion(Enum):
    """Enum type for POTCAR versions."""

    PBE_52 = "PBE_52"
    PBE_52_W_HASH = "PBE_52_W_HASH"
    PBE_54 = "PBE_54"
    PBE_54_W_HASH = "PBE_54_W_HASH"
    PBE_64 = "PBE_64"
    LDA_52 = "LDA_52"
    LDA_52_W_HASH = "LDA_52_W_HASH"
    LDA_54 = "LDA_54"
    LDA_54_W_HASH = "LDA_54_W_HASH"
    LDA_64 = "LDA_64"
    PW91 = "PW91"
    LDA_US = "LDA_US"
    PW91_US = "PW91_US"
    Perdew_Zunger81 = "Perdew_Zunger81"
```

could someone migrate this issue over to `pymatgen` unless there are other suggestions?

**Comment by esoteric-ephemera** [comment]
@janosh you may be misunderstanding the `RunType` enum: it's the functional used, not the pseudopotential type. That being said, it would be better for `pymatgen` to differentiate between the POTCAR versions used in the `Potcar` object, see related `pymatgen` issue [here](https://github.com/materialsproject/pymatgen/issues/3951)

Would be happy to do this on the pymatgen side

**Comment by janosh** [comment]
> That being said, it would be better for pymatgen to differentiate between the POTCAR versions used in the Potcar object

yeah exactly, i think having such an `Enum` would allow bringing clarity to the variety and subtle differences between POTCAR versions.

> Would be happy to do this on the pymatgen side

that would be much appreciated! 👍

**Comment by tsmathis** [comment]
okay, since this isn't an _error_ in emmet/the run_type enums I am going to move this to a discussion
